x-pack/plugins/viz_editor/index.ts

```diff
@@ -21,8 +21,8 @@ export function vizEditor(kibana: any) {
         title: 'Viz Editor',
         description: 'Explore and visualize data.',
         main: `plugins/${PLUGIN_ID}/app`,
-        icon: 'plugins/kibana/assets/visualize.svg',
-        euiIconType: 'visualizeApp',
+        // icon: 'plugins/kibana/assets/visualize.svg',
+        // euiIconType: 'visualizeApp',
```

Comment:
Commented these out so the sidebar would show the initials of the app instead of the same icon as Visualize.